sdk/search/azure-search/src/main/java/com/azure/search/implementation/models/SearchError.java

```diff
@@ -4,7 +4,7 @@
 // Changes may cause incorrect behavior and will be lost if the code is
 // regenerated.
 
-package com.azure.search.models;
+package com.azure.search.implementation.models;
```

Comment:
This is part of the public API for `SearchErrorException` and will need to remain under models, not implementation.models.

Reply:
Right, will move it back